src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [];
+
+export const booksSlice = createSlice({
+  name: 'storeBooks',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      const { id, title, author } = action.payload;
+      state.push({ id, title, author });
```

Comment:
- [Optional] Kindly use immutable array methods for updating state because it is best practice to use mutable methods in the reducer 👌
```
state.books = [
        ...state.books,
        {
          id: action.payload.id,
          title: action.payload.title,
          author: action.payload.author,
        },
      ]
```

Reply:
Hi @binhussen , thanks for the suggestion; but when I use this method I receive an error from eslint: `Assignment to property of function parameter 'state'`. How can I prevent this?